src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/EmailPreview/EmailPreview.tsx

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import type { FC } from 'react'
+import styled from 'styled-components'
+import Button from 'components/Button'
+import { themeColor, themeSpacing } from '@amsterdam/asc-ui'
+import ModalHeader from '../ModalHeader/ModalHeader'
+
+const ButtonWrapper = styled.div`
+  box-sizing: border-box;
```

Comment:
This rule should be applied to all elements by default. Is it needed?

Reply:
strange enough, yes.